src/LineupComponent1L.java

```diff
@@ -0,0 +1,130 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Convention: - `lineup` has only valid positions, which are: {"PG", "SG",
+ * "SF", "PF", "C"}. - Every player has their own PlayerBasics objects, which
+ * are their name and position.
+ *
+ * Correspondence: - `this` is the current starting lineup. - Key-value pairs in
+ * `lineup` equates to a position and its current starter.
+ */
+
+public class LineupComponent1L extends LineupComponentSecondary {
+
+    private Map<String, List<PlayerBasics>> lineup;
+
+    public LineupComponent1L() {
+        this.lineup = new HashMap<>();
+        this.lineup.put("PG", new ArrayList<>());
+        this.lineup.put("SG", new ArrayList<>());
+        this.lineup.put("SF", new ArrayList<>());
+        this.lineup.put("PF", new ArrayList<>());
+        this.lineup.put("C", new ArrayList<>());
+    }
+
+    private boolean isPosition(String pos) {
```

Comment:
Not required, but it's a good idea to mark your methods with `@Override` if they're overridden. (of course, this one is private as a helper I assume).